app/adapters/offer.js

```diff
@@ -0,0 +1,14 @@
+import ApplicationAdapter from "./application";
+import config from '../config/environment';
+
+export default ApplicationAdapter.extend({
+
+  urlForQuery: function(query, modelName){
+    const { slug } = query;
+    if (modelName === "offer" && slug === "search") {
+      return `${config.APP.API_HOST_URL}/${config.APP.NAMESPACE}/offers/search`;
```

Comment:
just curious, why don't we do the same as orders and use the `index` method ?

e.g `/offers?searchText=<text>&state=received`

Reply:
I guess index is already seems to be loaded with lots of logic may be `search` was written for separation of concerns.